### ISSUE TITLE
20582798: OS X VoiceOver: Screen Curtain Not Persistent Across Restarts

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#1457]
#### Description

Summary:
Screen curtain is not persistent across restarts of VoiceOver or the computer.

Steps to Reproduce:
1. Start VoiceOver.
1. Turn on Screen Curtain (VO+Shift+F11).
2. Turn off VoiceOver.
3. Turn on VoiceOver.
4. Steps 3 and 4 are automatic if the computer is restarted.

Expected Results:
Screen Curtain should be re-enabled when VoiceOver is started, provided that it was on for the user account when VoiceOver was last switched off.

Actual Results:
Screen Curtain doesn't persist; it's always off when VoiceOver is restarted, regardless of whether it was on when VoiceOver was last switched off.

Notes:
The intended behaviour described is that presently found in the version of VoiceOver in iOS.
## 

Product Version: OS X
Created: 2015-04-17T05:08:17.644350
Originated: 2015-04-17T06:00:00
Open Radar Link: http://www.openradar.me/20582798
